spiceaidocs/docs/features/ai-gateway/memory.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Language Model Memories'
```

Comment:
I think this should be a top-level feature. It applies to AI gateway and Models, so it's not a sub-feature of AI Gateway.